src/main/java/io/kestra/plugin/compress/ArchiveDecompress.java

```diff
@@ -36,11 +36,22 @@
 @Plugin(
     examples = {
         @Example(
-            code = {
-                "from: \"{{ inputs.files }}\"",
-                "algorithm: ZIP",
-                "compression: GZIP"
-            }
+            full = true,
+            code = """
+                id: archive_decompress
+                namespace: company.team
+
+                inputs:
+                  - id: file      #compressed file
```

Comment:
Can you add the comment as the input description instead? Think that will be clearer to the user and also reminds them that descriptions are a feature